### PR TITLE
Wine bump (5.0)

### PR DESCRIPTION
**Pull request by lovesegfault**
###### Motivation for this change
Well, guess I should've waited with yesterday's bump

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/misc/emulators/wine/sources.nix [hunk 1]
```diff
@@ -39,9 +39,9 @@ in rec {
 
   unstable = fetchurl rec {
     # NOTE: Don't forget to change the SHA256 for staging as well.
```
Comment (worldofpeace) [inline pkgs/misc/emulators/wine/sources.nix]: it looks like you didn't change the hash to staging

Comment (lovesegfault) [reply pkgs/misc/emulators/wine/sources.nix]: Fixed!

**Comment by lovesegfault** [comment]
@avnik @tadeokondrak Can y'all take a look? :)

**Review by avnik — APPROVED** [review]
LGTM

**Pull request merged**